src/pages/questions/QuestionDisplay.jsx

```diff
@@ -321,28 +354,23 @@ const QuestionDisplay = ({
           startIcon={<DoneIcon />}
           color="success"
           variant="contained"
-          size="large"
-          sx={{ display: "flex", flexDirection: "column", flexGrow: 1 }}
+          size="medium"
+          // sx={{ display: "flex", flexDirection: "column", flexGrow: 1 }}
+          sx={{ py: "1rem" }}
+          style={{
+        
+        maxWidth: '300px',
+        // margin: 'auto',
+        marginLeft: '5%'
+        
+    }}
+
         >
           {t("questions.yes")} ({shortcuts.yes})
         </Button>
       </Stack>
-      <Button
-        onClick={() =>
-          answerQuestion({
-            value: SKIPPED_INSIGHT,
-            insightId: question?.insight_id,
-          })
-        }
-        color="secondary"
-        variant="contained"
-        size="medium"
-        autoFocus
-        sx={{ py: "1rem" }}
-      >
-        {t("questions.skip")} ({shortcuts.skip})
-      </Button>
```

Comment:
I don't get how this modification is related to the issue you try to fix smae for others diff which have no link with packaging

Reply:
I mistakenly added the files which were previously edited for some other issue. I will rectify it as soon as possible. Sorry for the inconvenience.